src/Auxiliary/Aux_Check_Parameter.cpp

```diff
@@ -1093,8 +1120,47 @@ void Aux_Check_Parameter()
       Aux_Error( ERROR_INFO, "ELBDM_BASE_SPECTRAL must work with SUPPORT_FFTW !!\n" );
 #  endif
 
-#  if ( WAVE_SCHEME == WAVE_FD )
+// check hybrid scheme parameters for errors
+#  if ( ELBDM_SCHEME == ELBDM_HYBRID )
+
+   if ( INIT_SUBSAMPLING_NCELL > 1 )
+      Aux_Error( ERROR_INFO, "ELBDM_HYBRID currently does not support INIT_SUBSAMPLING_NCELL > 1 !!\n" );
+
+   if ( OPT__OUTPUT_TOTAL == 2 )
+      Aux_Error( ERROR_INFO, "ELBDM_HYBRID currently does not support OPT__OUTPUT_TOTAL == 2 !!\n" );
+
+#  if ( FLU_GHOST_SIZE < HYB_GHOST_SIZE )
+#  error : ERROR : FLU_GHOST_SIZE needs to be bigger than HYB_GHOST_SIZE !!
+#  endif // # if ( FLU_GHOST_SIZE < HYB_GHOST_SIZE )
 
+   if ( ELBDM_BASE_SPECTRAL )
+      Aux_Error( ERROR_INFO, "ELBDM_BASE_SPECTRAL incompatible with ELBDM_SCHEME == ELBDM_HYBRID !!\n" );
+
+   if ( MAX_LEVEL > 0 && FLAG_BUFFER_SIZE != PATCH_SIZE )
+      Aux_Error(  ERROR_INFO, "ELBDM_HYBRID with AMR requires that the flag buffer size equal the patch size!!\n");
```

Comment:
Just to be sure. Is it ok for `FLAG_BUFFER_SIZE_MAXM1_LV` and `FLAG_BUFFER_SIZE_MAXM2_LV` to be smaller than `PATCH_SIZE`? If so, replace `flag buffer size` with `FLAG_BUFFER_SIZE` in the error message to be more specific.